src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -3317,6 +3317,11 @@ private static void ThrowInvalidField(string f)
             result.LastQueryTime = _lastQueryingTime ?? DateTime.MinValue;
             result.ResultEtag = CalculateIndexEtag(queryContext, indexContext, q, result.IsStale);
             result.NodeTag = DocumentDatabase.ServerStore.NodeTag;
+            
+            if (q.SpatialResults != null)
+            {
+                result.SpatialProperties = q.SpatialResults.ToArray();
```

Comment:
Naming inconsistency - so those are sparial properties or results?

Reply:
done - renamed to `SpatialProperties`